README.md

```diff
@@ -2,20 +2,19 @@
 # github-rest-api-v3
 GitHub REST API playground
 
-https://adityabisoi.github.io/github-rest-api-v3/
+You can access it [here](https://adityabisoi.github.io/github-rest-api-v3/)
 
 ## Installation
 * Fork and clone the project to your local machine
-* `cd github-rest-api-v3/`
+* Move to the cloned repo's location `cd github-rest-api-v3/`
 * Run `npm install` to install ReactJS dependencies
 * Run `npm start` to run the project
 
 ## Documentation
-https://bit.ly/3kRkM2e
+You can refer to the documentation [here](https://bit.ly/3kRkM2e)
 
-## 🏆 Contributing
-
-Please read  [CONTRIBUTING.md](CONTRIBUTING.md)  for information on how to contribute to github-rest-api-v3.
+## Contributing 🏆
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) before contributing to the repository (github-rest-api-v3)
 
 ## Support
 <a href="https://gitter.im/github-rest-api-v3/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge" rel="some text">![Foo](https://icon-icons.com/icons2/2530/PNG/72/gitter_button_icon_151850.png
```

Comment:
Add an `Open Source Events` section mentioning the project being a part of NJHACK. Add NJHACK logo as well.

Reply:
@adityabisoi, I'm afraid I can't find anything about NJHACK, could you send me the relevant link(s)? I've added the outline of the subsection, but require further details to flesh it out

Reply:
Just add 'This project is a part of NJACK' with NJHACK pointing a link to the website https://njackwinterofcode.github.io/
And, add the logo
![nwoc-logo](https://user-images.githubusercontent.com/42573842/101052659-eb1b1600-35ac-11eb-9a80-cfaa33a29b04.png)

Reply:
How about now?

Reply:
Make it smaller, and centre it.
https://stackoverflow.com/questions/12090472/github-readme-md-center-image

Reply:
Done, I've set it at 200px